lib/components/search_user/interactor/mappers/search_user_state_mapper.dart

```diff
@@ -4,16 +4,31 @@ import 'package:seeds/domain-shared/page_state.dart';
 import 'package:seeds/domain-shared/result_to_state_mapper.dart';
 
 class SearchUserStateMapper extends StateMapper {
-  SearchUserState mapResultToState(SearchUserState currentState, Result result, List<String>? noShowUsers) {
-    if (result.isError) {
+  SearchUserState mapResultToState(
+      SearchUserState currentState, Result seedsMembersResult, Result telosResult, List<String>? noShowUsers) {
+    if (seedsMembersResult.isError && telosResult.isError) {
       return currentState.copyWith(pageState: PageState.failure, errorMessage: 'Error Searching for User');
     } else {
-      final List<MemberModel> users = result.asValue?.value as List<MemberModel>;
-      if (noShowUsers != null) {
-        for (final noShowUser in noShowUsers) {
-          users.removeWhere((element) => element.account == noShowUser);
+      List<MemberModel> seedsUsers = [];
+      List<MemberModel> telosUsers = [];
+
+      if (seedsMembersResult.isValue) {
+        seedsUsers = seedsMembersResult.asValue?.value as List<MemberModel>;
+        if (noShowUsers != null) {
+          for (final noShowUser in noShowUsers) {
+            seedsUsers.removeWhere((element) => element.account == noShowUser);
+          }
         }
       }
+
```

Comment:
if a telos account already appears up there in the seeds accounts - since all seeds accounts are also telos accounts - then we filter it out.

Reply:
This can be done cleaner. 

Telos user and seeds users are mapped down to `MemberModel`

If you override the equality function for member model, make it uniquely identifiable by the account

then you dont need to create multiple lists of accounts and what not.

Reply:
Well but I want to prefer Seeds users - Seeds users should not show up as Telos users. They're not interchangeable. Seeds users preferred.

Reply:
IDK there's probably a more elegant way to achieve these goals
- get list of Seeds users
- get list of Telos users, remove seeds users from that list
- remove no show users from the final list

We can't just to uniqueness, since Seeds users should not be changed by the Telos list, only the telos list should be changed by the Seeds list.